app/__init__.py

```diff
@@ -31,15 +32,26 @@ def create_app(config_name):
         login_manager=login_manager,
     )
 
+    def try_load_manifest(framework_data, question_set, manifest):
+        try:
+            content_loader.load_manifest(framework_data['slug'], question_set, manifest)
+
+        except ContentNotFoundError:
+            # Not all frameworks have this, so no need to panic (e.g. G-Cloud 4, G-Cloud 5, G-Cloud-6)
```

Comment:
This comment doesn't make much sense when this logic is abstracted to it's own function. If anything, the comment should go with where we decide to call `try_load_manifest`, defining why we think it's ok to use the function.